code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -359,4 +359,387 @@ Difficulty: Very Hard
+								/obj/item/projectile/energy/net, /obj/item/projectile/bullet/meteorshot, /obj/item/projectile/beam/xray)
+
+/obj/machinery/anomalous_crystal/emitter/ActivationReaction(mob/user, method)
+	if(..())
+		var/obj/item/projectile/P = new generated_projectile(get_turf(src))
+		P.setDir(dir)
+		switch(dir)
+			if(NORTH)
+				P.yo = 20
+				P.xo = 0
+			if(EAST)
+				P.yo = 0
+				P.xo = 20
+			if(WEST)
+				P.yo = 0
+				P.xo = -20
+			else
+				P.yo = -20
+				P.xo = 0
+		P.fire()
+
+/obj/machinery/anomalous_crystal/dark_reprise //Revives anyone nearby, but turns them into shadowpeople. Cannot revive shadowpeople, so this is a one time thing.
+	activation_method = "touch"
+
+/obj/machinery/anomalous_crystal/dark_reprise/ActivationReaction(mob/user, method)
+	if(..())
+		for(var/i in range(1, src))
+			if(ishuman(i))
+				var/mob/living/carbon/human/H = i
+				if(H.stat == DEAD && !isshadowperson(H))
+					H.set_species(/datum/species/shadow, 1)
+					H.revive(1,0)
+
+/obj/machinery/anomalous_crystal/helpers //Lets ghost spawn as helpful creatures that can only heal people slightly. Incredibly fragile and they can't converse with humans
+	activation_method = "touch"
+	var/ready_to_deploy = 0
+
+/obj/machinery/anomalous_crystal/helpers/ActivationReaction(mob/user, method)
+	if(..() && !ready_to_deploy)
+		ready_to_deploy = 1
+
+/obj/machinery/anomalous_crystal/helpers/attack_ghost(mob/dead/observer/user)
+	..()
+	if(ready_to_deploy)
+		var/be_helper = alert("Become a Lightgeist? (Warning, You can no longer be cloned!)",,"Yes","No")
+		if(!be_helper == "No")
+			return
+		var/mob/living/simple_animal/hostile/lightgeist/W = new /mob/living/simple_animal/hostile/lightgeist(get_turf(loc))
+		W.key = user.key
+
+/mob/living/simple_animal/hostile/lightgeist
+	name = "lightgeist"
+	desc = "This small floating creature is a completely unknown form of life... being near it fills you with a sense of tranquility."
+	icon_state = "lightgeist"
+	icon_living = "lightgeist"
+	icon_dead = "butterfly_dead"
+	turns_per_move = 1
+	response_help = "waves away"
+	response_disarm = "brushes aside"
+	response_harm = "disrupts"
+	speak_emote = list("oscillates")
+	maxHealth = 2
+	health = 2
+	harm_intent_damage = 1
+	friendly = "mends"
+	density = 0
+	flying = 1
+	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
+	ventcrawler = 2
+	mob_size = MOB_SIZE_TINY
+	gold_core_spawnable = 0
+	verb_say = "warps"
+	verb_ask = "floats inquisitively"
+	verb_exclaim = "zaps"
+	verb_yell = "bangs"
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	luminosity = 4
+	faction = list("neutral")
+	languages_spoken = 0
+	languages_understood = ALL
+	del_on_death = 1
+	minbodytemp = 0
+	maxbodytemp = 1500
```

Comment:
This does nothing if their unsuitable_atmos_damage is 0, because even if exceeded they'll take 0 damage.